### ISSUE TITLE
Error when trying to link # with missing dir

### ISSUE THREAD

**Issue by abrudz** [Dyalog/link#181]
**To Reproduce**
```
      ]create # /tmp/newdir
* Command Execution Failed: ⎕SE.Link.Create: Cannot get source of #. : invalid name class (1)
```
**Expected behaviour**
Create dir and link.

**Desktop (please complete the following information):**
 - OS: Windows
 - APL version: 19.0.39491u64
 - `⎕SE.Link.Version`: 2.1.0-beta5

**Additional context**
`]create # /tmp/newdir -s=ns` has same effect.

**Comment by nicolas-dyalog** [comment]
I believe it's fixed in v2.1.0-beta18 - can you pull it from the master branch and confirm that the bug is gone ?

**Comment by abrudz** [comment]
Fixed.
